spellcast.py

```diff
@@ -88,6 +97,19 @@ def __init__(self):
         Reads words from a file and initializes various attributes needed for
         managing the game board.
         """
+        self.words = set()
+        self.letter_values = {}

```

Comment:
`letter_values` can probably be moved to a local scope in recalculate.

Reply:
Scoped.